extensions/cache/runtime/pom.xml

```diff
@@ -24,6 +24,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-caffeine</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.microprofile.context-propagation</groupId>
+            <artifactId>microprofile-context-propagation-api</artifactId>
+        </dependency>
```

Comment:
@FroMage @stuartwdouglas what's your opinion on having a dependency on that one here? It's a bit related to what you are currently doing with the other PR.

Reply:
If that can help answer @gsmet's question, this dependency is only present here to allow the `ManagedExecutor` import and usage in `CaffeineCacheBuildRecorder`.